### PR TITLE
Kits: Change - Add confirmation message before move kit to trash

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/kits/manager.php [hunk 1]
```diff
@@ -70,7 +83,7 @@ public function get_current_settings( $setting = null ) {
 	private function create_default() {
 		$kit = Plugin::$instance->documents->create( 'kit', [
 			'post_type' => Source_Local::CPT,
-			'post_title' => __( 'Default Kit', 'elementor' ),
+			'post_title' => __( 'Default Site Settings', 'elementor' ),
```
Comment (matipojo) [inline core/kits/manager.php:86]: Remove. Currently we won't update the name.

File: core/kits/manager.php [hunk 2]
```diff
@@ -262,6 +275,35 @@ private function add_body_class() {
...
+	 * @param false $is_permanently_delete
+	 */
+	private function before_delete_kit( $post_id, $is_permanently_delete = false ) {
+		if (
+			! $this->is_kit( $post_id ) ||
+			isset( $_GET['force'] ) ||  // phpcs:ignore -- nonce validation is not require here.
+			(
+				$is_permanently_delete &&
+				( ! defined( 'EMPTY_TRASH_DAYS' ) || 0 !== EMPTY_TRASH_DAYS )
+			)
```
Comment (matipojo) [inline core/kits/manager.php:292]: Extract to a variable.

File: core/kits/manager.php [hunk 3]
```diff
@@ -262,6 +275,35 @@ private function add_body_class() {
...
+	 * Send a confirm message before move a kit to trash, or before permanently delete
+	 * when the const EMPTY_TRASH_DAYS = 0
+	 *
+	 * @param       $post_id
+	 * @param false $is_permanently_delete
+	 */
+	private function before_delete_kit( $post_id, $is_permanently_delete = false ) {
+		if (
+			! $this->is_kit( $post_id ) ||
+			isset( $_GET['force'] ) ||  // phpcs:ignore -- nonce validation is not require here.
```
Comment (matipojo) [inline core/kits/manager.php:288]:
```suggestion
			isset( $_GET['force_delete_kit'] ) ||  // phpcs:ignore -- nonce validation is not require here.
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/kits/manager.php (hunk 2 above)
Comment (Nevoss) [reply core/kits/manager.php]: I changed the logic.

File: core/kits/views/trash-kit-confirmation.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+<?php
+if ( $is_permanently_delete ) {
+	echo __( 'If you delete this template, all associated global settings, typography and colors will be deleted', 'elementor' );
+} else {
+	echo __( 'If you move this template to trash, all associated global settings, typography and colors will be deleted', 'elementor' );
+}
+?>
+
+<br><br>
+<a class="button" href="<?php echo add_query_arg( [ 'force_delete_kit' => '1' ], get_delete_post_link( $post_id, '', $is_permanently_delete ) ); ?>">
```
Comment (matipojo) [inline core/kits/views/trash-kit-confirmation.php:21]: Extract to a variable.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**